Lesson5/src/Military.java

```diff
@@ -0,0 +1,57 @@
+public class Military extends airTransport {
```

Comment:
airTransport -> AirTransport. All class names should start from the capital letter